### ISSUE TITLE
Module option `pages` doesn't create translated routes

### ISSUE THREAD

**Issue by bstnhnsl** [leanera/nuxt-i18n#1]
### Versions

nuxt: 3.0.0-rc.11
nuxt-i18n: 0.3.8

### Setup

Pages folder:

```
nuxt
└───pages
│   │   index.vue
│   │   projects.vue
```

nuxt.config.ts:

```i18n: {
    locales: ['de', 'en', 'fr'],
    defaultLocale: 'de',
    strategy: 'prefix_except_default',
    pages: {
      projects: {
        de: '/projekte', 
        en: '/projects', 
        fr: '/projects', 
      }
    }
```

### What is Expected?

create route for the default 'de' path '/projekte'

### What is actually happening?

404 on path '/projekte'
The route '/projekte' isn't created at all

Created routes:
```
{
    name: 'projects___de',
    path: '/projects',
    file: '/nuxt/pages/projects.vue',
    children: []
  },
  {
    name: 'projects___en',
    path: '/en/projects',
    file: '/nuxt/pages/projects.vue',
    children: []
  },
  {
    name: 'projects___fr',
    path: '/fr/projects',
    file: '/nuxt/pages/projects.vue',
    children: []
  }
```
  
  ### Workaround
  
  I used the routeOverrides option to get the translated routes/paths

**Comment by johannschopplich** [comment]
Hi there Bastian,
thanks for the verbose issue description! Indeed, the `pages` option wasn't working properly. Background: Instead of matching a route's name as the identifier for the `pages` option keys, the route path was used. 

Fixed in [0.3.10](https://github.com/leanera/nuxt-i18n/releases/tag/v0.3.10). Could you test again, please?

**Comment by johannschopplich** [comment]
I consider the bug fixed. If it persists, please open a new issue. Thank you!
